R/digest_to_checksum.R

```diff
@@ -1,89 +1,123 @@
-#' Computes an MD5 checksum from columns
+#' Computes an checksum from columns
+#'
+#' @details
+#'   In most cases, the md5 algorithm is used to compute the checksums.
+#'   For Microsoft SQL Server, the SHA-256 algorithm is used.
 #'
 #' @name digest_to_checksum
 #'
 #' @template .data
 #' @param col Name of the column to put the checksums in
```

Comment:
```suggestion
#' @param col Name of the column to put the checksums in (Default: "checksum")
```

Reply:
Marcus and I have been omitting to write the default values in the param descriptions since they are clear in the function definition.

The nice thing about having it only there is that if this should change at any point, we do not risk forgetting to update the param description also.

Reply:
I can see that we have not consistently done so throughout the package, but I think it is best practice.

Reply:
Perfectly fine to not include